testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OfflineTokenTest.java

```diff
@@ -281,7 +281,7 @@ public void offlineTokenBrowserFlow() throws Exception {
     }
 
     private String testRefreshWithOfflineToken(AccessToken oldToken, RefreshToken offlineToken, String offlineTokenString,
-                                               final String sessionId, String userId) {
+                                               final String sessionId, String userId, boolean scopeParameterExist) {
```

Comment:
Since you removed the unused method, is it possible to remove the parameter `scopeParameterExist` from this method signature? Looks that it is not needed as it is always invoked with value `false` .

Reply:
Now, I have remembered my goal of no used removed method.

My goal was to test refresh token request with scope parameter without offline_access and refresh token containing offline_access. However, I have never used this method. See my last commit. What is your opinion? I wait also for test running to be sure.

Reply:
If you want to add the new test for the scenario you mentioned, feel free to do that.

But if you prefer to not adding that scenario, it would be good to remove the parameter as I've pointed above. IMO it is not strictly needed to add the test for the scenario you mentioned, but whatever you prefer.

Reply:
See my last comment